app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<h1> Location for <strong><%= @booking.star.name %></strong> </h1>
+<h3> <%= (@booking.star.price * (@booking.end_date - @booking.start_date)).to_i %> €</h3>
+<p> <%= @booking.start_date %>  to  <%= @booking.end_date %></p>
+
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<%= simple_form_for [ @booking, @review ] do |f| %>
+  <%= f.input :rating, :required => false %>
+  <%= f.input :comment, :required => false %>
+  <%= f.submit "Add a review" %>
+<% end %>
```

Comment:
Do we need a booking page if we have a my_bookings and a my_listings ?

Reply:
yes because it's a show page and not an index page